tests/test_api.py

```diff
@@ -71,6 +71,20 @@ def test_update_stores_checkout_value(value, tmpdir):
     assert json.load((project_dir / ".cruft.json").open("r"))["checkout"] == value
 
 
+def test_fail_on_non_unicode_in_diff(tmpdir):
+    tmpdir.chdir()
+    project_dir = cruft.create(
+        "https://github.com/tdhopper/cookiecutter-test",
```

Comment:
You can use https://github.com/cruft/cookiecutter-test to create any test data btw.

Reply:
@samj1912 What's the process for getting a new branch on there?

Reply:
I can temporarily add you to the repository. One second.

Reply:
Sent!